### ISSUE TITLE
Facets - Color Contrast

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#518]
SC 1.4.3 AA – Contrast – “Maps” and “21” under Resource Class section has insufficient color contrast
of 3.2 (foreground color: #ffffff, background color: #28A644)

**Comment by ewlarson** [comment]
^ For active search constraint state facets
